### PR TITLE
Feature/OE-582 ratios calculated incorrectly

### PR DESCRIPTION
**Pull request by ron-junior-devsquad**
## Review

For the submitter, initial self-review:

- [ ] Reviewed the test cases added for completeness and possible points for discussion
- [ ] Check the scope of the changeset - is everything in the diff required for the pull request?

For the pull request reviewer(s), this changeset has been reviewed for:

- [ ] Consistency between the changeset and the goal stated above
- [ ] Usage friction - is the proposed change in usage cumbersome or complicated?
- [ ] Complexity - is the proposed change introducing any unjustified complexity? 
- [ ] Thoroughness of added tests and any missing edge cases

### CODE REVIEW

File: app/Http/Livewire/NumberTracker/NumbersRatios.php [hunk 1]
```diff
@@ -80,7 +80,7 @@ public function getSitRatiosProperty()
             $satsSum = $this->numbers->sum('sats');
 
             return $satsSum > 0
-                ? number_format($this->numbers->sum('sets') / $satsSum, 2)
+                ? number_format($satsSum / $this->numbers->sum('sets'), 2)
```
Comment (devsquad-mathews-lima) [inline app/Http/Livewire/NumberTracker/NumbersRatios.php:83]: I think here is possible to occur a division by zero.

File: app/Http/Livewire/NumberTracker/NumbersRatios.php [hunk 2]
```diff
@@ -91,7 +91,7 @@ public function getCloseRatioProperty()
             $closesSum = $this->numbers->sum('closes');
 
             return $closesSum > 0
-                ? number_format($this->numbers->sum('closer_sits') / $closesSum, 2)
+                ? number_format($closesSum / $this->numbers->sum('closer_sits'), 2)
```
Comment (devsquad-mathews-lima) [inline app/Http/Livewire/NumberTracker/NumbersRatios.php:94]: I think here is possible to occur a division by zero.

File: app/Http/Livewire/Scoreboard.php [hunk 3]
```diff
@@ -84,11 +84,11 @@ public function setUser($userId)
 
             $this->dpsRatio = $dailyNumbers->sum('doors') / $dailyNumbers->sum('sets');
             $this->hpsRatio = (float)$dailyNumbers->sum('hours_knocked') / $dailyNumbers->sum('sets');
-            $this->sitRatio = $satsSum > 0 ? ($dailyNumbers->sum('sets') / $satsSum) : 0;
+            $this->sitRatio = $satsSum > 0 ? ($satsSum / $dailyNumbers->sum('sets')) : 0;
```
Comment (devsquad-mathews-lima) [inline app/Http/Livewire/Scoreboard.php:87]: I think here is possible to occur a division by zero.

File: app/Http/Livewire/Scoreboard.php [hunk 4]
```diff
@@ -84,11 +84,11 @@ public function setUser($userId)
...
         }
 
         if ($dailyNumbers->sum('closes') > 0) {
-            $this->closeRatio = $dailyNumbers->sum('closer_sits') / $dailyNumbers->sum('closes');
+            $this->closeRatio = $dailyNumbers->sum('closes') / $dailyNumbers->sum('closer_sits') ;
```
Comment (devsquad-mathews-lima) [inline app/Http/Livewire/Scoreboard.php:91]: I think here is possible to occur a division by zero.

File: resources/views/components/profile/show-profile-information.blade.php [hunk 5]
```diff
@@ -73,7 +73,7 @@
             </div>
             <div class="col-span-4 text-xl font-bold text-gray-900">
                 @if(user()->dailyNumbers->sum('sats') > 0)
-                    {{ number_format(user()->dailyNumbers->sum('sets') / user()->dailyNumbers->sum('sats'), 2) }}%
+                    {{ number_format( user()->dailyNumbers->sum('sats') / user()->dailyNumbers->sum('sets'), 2) }}%
```
Comment (devsquad-mathews-lima) [inline resources/views/components/profile/show-profile-information.blade.php:76]: I think here is possible to occur a division by zero.

File: resources/views/components/profile/show-profile-information.blade.php [hunk 6]
```diff
@@ -85,7 +85,7 @@
             </div>
             <div class="col-span-4 text-xl font-bold text-gray-900">
                 @if(user()->dailyNumbers->sum('closes') > 0)
-                    {{ number_format(user()->dailyNumbers->sum('closer_sits') / user()->dailyNumbers->sum('closes'), 2) }}%
+                    {{ number_format(user()->dailyNumbers->sum('closes') / user()->dailyNumbers->sum('closer_sits'), 2) }}%
```
Comment (devsquad-mathews-lima) [inline resources/views/components/profile/show-profile-information.blade.php:88]: I think here is possible to occur a division by zero.

Comment (devsquad-mathews-lima) [inline resources/views/components/profile/show-profile-information.blade.php:88]: I think here is possible to occur a division by zero.

File: resources/views/components/profile/show-profile-information.blade.php (hunk 5 above)
Comment (devsquad-mathews-lima) [inline resources/views/components/profile/show-profile-information.blade.php:76]: I think here is possible to occur a division by zero.

File: app/Http/Livewire/Scoreboard.php (hunk 4 above)
Comment (devsquad-mathews-lima) [inline app/Http/Livewire/Scoreboard.php:91]: I think here is possible to occur a division by zero.

File: app/Http/Livewire/Scoreboard.php (hunk 3 above)
Comment (devsquad-mathews-lima) [inline app/Http/Livewire/Scoreboard.php:87]: I think here is possible to occur a division by zero.

File: app/Http/Livewire/NumberTracker/NumbersRatios.php (hunk 2 above)
Comment (devsquad-mathews-lima) [inline app/Http/Livewire/NumberTracker/NumbersRatios.php:94]: I think here is possible to occur a division by zero.

File: app/Http/Livewire/NumberTracker/NumbersRatios.php (hunk 1 above)
Comment (devsquad-mathews-lima) [inline app/Http/Livewire/NumberTracker/NumbersRatios.php:83]: I think here is possible to occur a division by zero.

**Review by devsquad-mathews-lima — CHANGES_REQUESTED** [review]

**Review by devsquad-mathews-lima — APPROVED** [review]
